scripts/scr_marine_struct/scr_marine_struct.gml

```diff
@@ -9,89 +32,185 @@ enum location_types {
 	warp
 }
 global.trait_list = {
+	"champion":{
+		weapon_skill : 20,
+		ballistic_skill:20,
+		display_name : "Champion",
+		flavour_text : "Through either natural talent, or obsessive training {0} is a master of arms"
+	},
+	"lightning_warriors":{
+		constitution: -6,
+		dexterity :6,
+		weapon_skill : 5,
+		flavour_text : "{0} is a master of speed covering distances quickly to enter the fray",
+		display_name : "Lightning Warrior",
+	},
+	"slow_and_purposeful":{
+		constitution:6,
+		dexterity : -6,
+		strength : 2,
+		flavour_text : "{0} is implacable advancing in combat with methodical reason",
+		display_name : "Slow and Purposeful",
+
+	},
+	"melee_enthusiast":{
+		weapon_skill : 5,
+		strength : 3,		
+		flavour_text : "nothing can keep {0} from the fury of close up battle",	
+		display_name : "Melee Enthusiast",
+
+	},
 	"shitty_luck":{
-		luck:-4
+		luck:-4,
+		flavour_text : "for all their talent {0} is dogged by poor luck",
+		display_name : "Shitty Luck",
 	},
 	"very_hard_to_kill":{
-		constitution:10,
+		constitution:15,
 		luck:2,
+		flavour_text : "{0} is possed of a toughness and luck unsurpassed by most",
+		display_name : "Very Hard To Kill",
+		
 	},
 	"paragon":{
-		constitution:6,
+		constitution:12,
 		luck:2,
-		strength:2,
-		dexterity:2,
-		intelligence:2,
-		charisma:2
+		strength:6,
+		dexterity:6,
+		intelligence:6,
+		charisma:6,
+		weapon_skill:10,
+		ballistic_skill:10,
+		flavour_text : "{0} walks in the footsteps of the primarchs of old",
+		display_name : "Paragon",
+
 	},
 	"warp_touched":{
-		intelligence:4
-	}
+		intelligence:4,
+		flavour_text : "has phychic potential even if unknown to them",
+		display_name : "Warp Touched",		
+	},
+	"lucky":{
+		luck : 4,
+		flavour_text : "Is inexplicably lucky",
+		display_name : "Lucky"
+	},
+	"old_guard":{
+		luck : 1,
+		constitution : 1,
+		strength :1,
+		weapon_skill : 2,
+		ballistic_skill :2,
+		flavour_text : "{0} has seen many a young warrior rise and die before him but he remains",
+		display_name : "Old Guard"
+	},
+	"seasoned":{
+		luck : 1,
+		constitution : 1,
+		strength :1,
+		weapon_skill : 1,
+		ballistic_skill :1,
+		flavour_text : "{0} has seen many a young warrior rise and die before him but he remains",
+		display_name : "Old Guard",		
+	},
+	"seasoned":{
+		luck : 1,
+		constitution : 1,
+		strength :1,
+		weapon_skill : 1,
+		ballistic_skill :1,
+		flavour_text : "{0} is a seasoned warrior having fought for many years",
+		display_name : "Seasoned",		
+	},
+	"ancient":{
+		luck : 1,
+		constitution : -1,
+		strength :-1,
+		weapon_skill : 3,
+		ballistic_skill :4,
+		wisdom : 5,
+		flavour_text : "{0} is truly Ancient while his body may ache his skills and wisdom are to be respected",
```

Comment:
truly Ancient.  **W**hile his body may ache, his ...

Reply:
yeah sorry will do a rerun of flavour and dialogue stuff in future